src/app/modules/ix-table2/components/ix-table-body/cells/ix-cell-text/ix-cell-text.component.html

```diff
@@ -1 +1 @@
-{{ value }}
+{{ value || ' - ' }}
```

Comment:
Noticed that this change will affect all the falsy values. Examples of affected values are `false`, `""`, `0`, `null`, `undefined`. And difference between `""` and `undefined` (or between `0` and `null`) could play significant role in some configurations. I suggest to use a more strict check to display only for empty values (`null` or `undefined`)

Reply:
> Noticed that this change will affect all the falsy values. Examples of affected values are `false`, `""`, `0`, `null`, `undefined`. And difference between `""` and `undefined` (or between `0` and `null`) could play significant role in some configurations. I suggest to use a more strict check to display only for empty values (`null` or `undefined`)

okay, make sense